tools/packaging/kata-deploy/scripts/kata-deploy.sh

```diff
@@ -302,6 +343,16 @@ function install_artifacts() {
 					;;
 			esac	
 		fi
+
+		if [ -n "${INSTALLATION_PREFIX}" ]; then
+			# We could always do this sed, regardless, but I have a strong preference
+			# on not touching the configuration files unless extremelly needed
+			sed -i -e "s|${default_dest_dir}|${dest_dir}|g" "${kata_config_file}"
+
+			# Let's only adjust qemu_cmdline for the QEMUs that we build and ship ourselves
+			[[ "${shim}" =~ ^(qemu|qemu-snp|qemu-nvidia-gpu|qemu-nvidia-gpu-snp|qemu-sev|qemu-se)$ ]] && \
```

Comment:
This feels like a bit of a trap that we might fall in in missing this if we add new runtime classes? I have no suggestion of how to remedy this, but it might be something to bear in mind

Reply:
Ideally we could do a simple `qemu*` check, but I thought it'd be better to make it explicit.
Let me know what's your preference.

Reply:
Yeah, I like the explicit check to rule out the chances of us overstepping, but does mean that there is a chance of missing something. Given that we don't necessarily expect lots of users to use different install directory, I think it's fine as is, but would welcome views from others

Reply:
Do we have a place that lists the QEMU based runtime classes in an authoritative way ? If so, we could have some helper function that checks for all of them.

Also, does this apply to stratovirt as well since it is supposedly a drop-in replacement for QEMU ?

Reply:
> Do we have a place that lists the QEMU based runtime classes in an authoritative way ? If so, we could have some helper function that checks for all of them.

We don't, anywhere. :-/

> Also, does this apply to stratovirt as well since it is supposedly a drop-in replacement for QEMU ?

I sincerely have zero idea about stratovirt.